README.md

```diff
@@ -31,28 +31,77 @@ using Aqua
 Aqua.test_all(YourPackage)
 ```
 
-## Notes on Aqua 0.6
+## How to add Aqua.jl...
 
-Aqua 0.6 includes the type piracy detection, thanks to [the PR](https://github.com/JuliaTesting/Aqua.jl/pull/88) by Jakob
-Nybo Nissen (@jakobnissen) and [the original implementation](https://discourse.julialang.org/t/pirate-hunter/20402) by
-Frames Catherine White (@oxinabox).
+### ...as a test dependency?
 
-If this part of Aqua 0.6 causes a trouble, then you can disable the piracy detection
-with a flag as in `Aqua.test_all(YourPackage; piracy = false)`.
+There are two ways to add Aqua.jl as a test dependency to your package.
+To avoid breaking tests when a new Aqua.jl version is released, it is
+recommended to add a version bound for Aqua.jl.
 
-## Specifying Aqua version
+ 1. In `test/Project.toml`, add Aqua.jl to `[dep]` and `[compat]` sections, like
+    ```toml
+    [deps]
+    Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
+    Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
-To avoid breaking test when a new Aqua.jl version is released, it is
-recommended to add version bound for Aqua.jl in `test/Project.toml`:
+    [compat]
+    Aqua = "0.6"
+    ```
 
-```toml
-[deps]
-Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+ 2. In `Project.toml`, add Aqua.jl to `[compat]` and `[extras]` section and the `test` target, like
+    ```toml
+    [compat]
+    Aqua = "0.6"
 
-[compat]
-Aqua = "0.6"
+    [extras]
+    Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
+    Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+    [targets]
+    test = ["Aqua", "Test"]
+    ```
+
+If your package supports Julia pre-1.2, you need to use the second approach, 
+although you can use both approaches at the same time.
+
+!!! warning
+    In normal use, `Aqua.jl` should not be added to `[deps]` in `Project.toml`!
```

Comment:
```suggestion
> **Warning**\
> In normal use, `Aqua.jl` should not be added to `[deps]` in `YourPackage/Project.toml`!
```
This suggestion changes the warning from
![image](https://github.com/JuliaTesting/Aqua.jl/assets/7488140/c5c507d2-7d1b-4a61-8d8b-ae795a4d5f72)
to
![image](https://github.com/JuliaTesting/Aqua.jl/assets/7488140/044d5d59-60c5-4fdd-9112-6339dd19618a)

Reply:
This unfortunately does not look good in the automatically generated docstring.

Reply:
The `!!!` variant is fine, it is an "admonition", see https://docs.julialang.org/en/v1/stdlib/Markdown/#Admonitions

Reply:
It wasn't fine as long as it was in the `README.md`, but since this changed, I think this comment is obsolete.

Reply:
Ah, I see. I overlooked that the documentation refers to README.md.